pkg/monitor/format/format.go

```diff
@@ -249,6 +249,8 @@ func (m *MonitorFormatter) FormatSample(data []byte, cpu int) {
 		m.policyVerdictEvents(prefix, data)
 	case monitorAPI.MessageTypeRecCapture:
 		m.recorderCaptureEvents(prefix, data)
+	case monitorAPI.MessageTypeTraceSock:
+		// TODO: events to be formatted
```

Comment:
Is this for outputting in `cilium monitor`? I wonder if it's possible to implement this with just a few lines of code and be able to view these events?

Reply:
PR to add a monitor parser - https://github.com/cilium/cilium/pull/21516.